bpf/Makefile

```diff
@@ -86,8 +84,8 @@ MAX_BASE_OPTIONS = -DSKIP_DEBUG=1 -DENABLE_IPV4=1 -DENABLE_IPV6=1 \
 	-DPOLICY_VERDICT_NOTIFY=1 -DALLOW_ICMP_FRAG_NEEDED=1 -DENABLE_IDENTITY_MARK=1 \
 	-DMONITOR_AGGREGATION=3 -DCT_REPORT_FLAGS=0x0002 -DENABLE_HOST_FIREWALL=1 \
 	-DENABLE_ICMP_RULE=1 -DENABLE_CUSTOM_CALLS=1 -DENABLE_SRV6=1 -DENABLE_L7_LB=1
-ifeq ("$(KERNEL)","49")
-# IPSec is incompatible with BPF NodePort so we only enable on 4.9.
+ifeq ("$(KERNEL)","419")
+# IPSec is incompatible with BPF NodePort so we only enable on 4.19.
 MAX_BASE_OPTIONS += -DENABLE_IPSEC=1 -DIP_POOLS=1
```

Comment:
Let's just drop this special case. The Makefile is just about generating the largest BPF programs possible and KPR is clearly impacting complexity more than IPsec.

Longer term we probably want to cover IPsec in complexity tests themselves (i.e., `bpf/complexity-tests/`) but that's orthogonal to this PR.